### PR TITLE
fix(DB/Creature): Change humans emote to sit/sleep

### PR DESCRIPTION
**Pull request by adame1181**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15610

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [x] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

wow1
wow2
wow3
wow4
wow5
wow6

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+(1970927, 0, 0, 3, 0, 0, 0, NULL),
+(1970877, 0, 0, 3, 0, 0, 0, NULL);
+
+-- UNCLICKABLE CHAIRS:
+UPDATE `gameobject_template` SET `type` = 5, `Data0` = 0, `Data1` = 0 WHERE `entry` IN
+(187521, 187515, 187514, 187512, 187519, 187522, 187523);
+
+-- UNCLICKABLE BED:
+UPDATE `gameobject_template` SET `type` = 5, `Data0` = 0, `Data1` = 0 WHERE
+(`entry` = 187526 );
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql:21]: `gameobject_template` is a sniffed table and cannot be changed without sniffs to back up the changes.

Comment (Kitzunu) [inline data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql:21]: Thanks for the PR, but please read through https://www.azerothcore.org/wiki/sql-standards and adjust the file accordingly.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql:21]: Just as Bench said, gamobject_template is a table that requires verification from sniffs if it should be changed.

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql (hunk 1 above)
Comment (heyitsbench) [reply data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql:21]: For the record, sniffs disagree with both template changes. What is in AC's latest DB (at least regarding these columns) lines up with 51943.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/fix-humans-emotes-in-culling-of-stratholme.sql:21]: Okay so delete these changes to the template entirely

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**